samples/bot-calling-meeting/csharp/Source/CallingBotSample/Bots/MessageBot.cs

```diff
@@ -54,55 +69,42 @@ protected override async Task OnMessageActivityAsync(ITurnContext<IMessageActivi
                 {
                     string type = value["type"];
                     type = string.IsNullOrEmpty(type) ? "." : type.ToLower();
-                    await SendResponse(turnContext, type, cancellationToken);
+                    string? callId = value["callId"] ?? null;
+                    await SendResponse(turnContext, type, callId, cancellationToken);
                 }
             }
             else
             {
                 turnContext.Activity.RemoveRecipientMention();
-                await SendResponse(turnContext, turnContext.Activity.Text.Trim().ToLower(), cancellationToken);
-            }
-        }
-
-        protected override async Task OnTeamsMeetingStartAsync(MeetingStartEventDetails meeting, ITurnContext<IEventActivity> turnContext, CancellationToken cancellationToken)
-        {
-            await SendMeetingActionsCard(turnContext, cancellationToken);
-        }
-
-        protected override async Task OnTeamsMeetingEndAsync(MeetingEndEventDetails meeting, ITurnContext<IEventActivity> turnContext, CancellationToken cancellationToken)
-        {
-            var conversationStateAccessors = conversationState.CreateProperty<MeetingActionDetails>(nameof(MeetingActionDetails));
-            var conversationData = await conversationStateAccessors.GetAsync(turnContext, () => new MeetingActionDetails());
-
-            var activity = MessageFactory.Text("This meeting has completed, you can no longer perform actions.");
-            activity.Id = conversationData.MeetingActionsCardActivityId;
-            if (activity.Id != null)
-            {
-                await turnContext.UpdateActivityAsync(activity, cancellationToken);
+                await SendResponse(turnContext, turnContext.Activity.Text.Trim().ToLower(), null, cancellationToken);
             }
         }
 
         protected override Task<TaskModuleResponse> OnTeamsTaskModuleFetchAsync(ITurnContext<IInvokeActivity> turnContext, TaskModuleRequest taskModuleRequest, CancellationToken cancellationToken)
         {
             var asJobject = JObject.FromObject(taskModuleRequest.Data);
-            var action = asJobject.ToObject<TaskModuleFetchData>()?.Action;
+            var fetchData = asJobject.ToObject<TaskModuleFetchData>();
 
             var taskInfo = new TaskModuleTaskInfo();
 
-            switch (action)
+            switch (fetchData?.Action)
             {
                 case "createcall":
-                    taskInfo.Card = adaptiveCardFactory.CreatePeoplePickerCard("Choose who to create a call with:", "Create", isMultiSelect: true);
+                    taskInfo.Card = adaptiveCardFactory.CreatePeoplePickerCard("Choose who to create a call with:", "Create", null, isMultiSelect: true);
```

Comment:
Add comments describing these cases, similar to the calling bot notification switch.